autosklearn/metalearning/mismbo.py

```diff
@@ -4,17 +4,10 @@
 
 from autosklearn.metalearning.optimizers.metalearn_optimizer.metalearner \
     import MetaLearningOptimizer
-
-from autosklearn.constants import *
+from autosklearn.constants \
+    import MULTILABEL_CLASSIFICATION, MULTICLASS_CLASSIFICATION, TASK_TYPES_TO_STRING
 from autosklearn.util.logging_ import get_logger
 
-__all__ = [
```

Comment:
I deleted those guys.  Let me know if this was not a good idea.

Reply:
If the code still works, it means they were unnecessary, so I'm fine with this.